labs/devnet-alexa-lab/3.md

```diff
@@ -1,4 +1,4 @@
-# Step 3: Create the Lamdba Function
+# Step 3: Create the lamdba function
```

Comment:
I believe Lamba is a proper name - so keep it capitalized. Same as Bucket I believe.